subsys/net/lib/openthread/platform/openthread-core-zephyr-config.h

```diff
@@ -14,6 +14,7 @@
 #define OPENTHREAD_CORE_ZEPHYR_CONFIG_H_
```

Comment:
Typo in the commit msg, `s/bulids/builds/`

Reply:
Fixed, thanks